ios/FluentUI/Table View/TableViewCell.swift

```diff
@@ -1396,9 +1568,7 @@ open class TableViewCell: UITableViewCell {
     }
 
     private func updateSelectionImageColor() {
-        if let window = window {
-            selectionImageView.tintColor = isSelected ? Colors.primary(for: window) : Colors.Table.Cell.selectionIndicatorOff
-        }
+        selectionImageView.tintColor = isSelected ? UIColor(dynamicColor: GlobalTokens().brandColors[.primary]) : UIColor(dynamicColor: tokens.selectionIndicatorOffColor)
```

Comment:
shouldn't the global tokens reference come from the tokens class?

**tokens.globalTokens**.brandColors[.primary] instead of using **GlobalTokens()**